jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -175,6 +175,64 @@ export class SortingViewModel extends DOMWidgetModel {
   static view_module_version = MODULE_VERSION;
 }
 
+interface PluginComponentWrapperProps {
+  plugin: SortingViewPlugin
+  hither: HitherContext
+  sortingObject: any
+  recordingObject: any
+  sortingInfo: any
+  recordingInfo: any
+}
+
+const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = ({plugin, hither, sortingObject, recordingObject, sortingInfo, recordingInfo}) => {
```

Comment:
It was necessary to make this wrapper component in order to have a useReducer() for the curation/curationDispatch